### ISSUE TITLE
Allow data_split_shuffle equal false with non-stratified sampling in Classification function

### ISSUE THREAD

**Issue by ykh96** [pycaret/pycaret#557]
Hi Team,

Just noticed that that for the classification problem pycaret will always use stratified sampling which will shuffle the data and cause problem when we set data_split_shuffle = False. The data shuffling can cause information leakage for timeseries classification. Just wondering if we can add an option whether to use the stratified sampling in the data setup so we can ban the shuffle. Hopefully will not take too much time to fix it. Thank u a lot for your help!

**Comment by pycaret** [comment]
@ykh96 Good point. Will take this in future release.

**Comment by pycaret** [comment]
@Yard1 Feel free to delegate this to anybody you would like.

@ykh96 Would you be interested in working on this?

**Comment by Yard1** [comment]
Isn't this related to adding other CV options?

**Comment by ykh96** [comment]
It's the train_test_split in the setup() function, it seems that we always do the stratified sampling on y, which force the shuffle=True

**Comment by Yard1** [comment]
Oh, alright. Thanks for clearing that up.

**Comment by pycaret** [comment]
@ykh96 That's correct. We are unfortunately forcing `stratify` param of `train_test_split` to target variable and hence `shuffle` is forced to `True`. In `2.2` release we will be introducing two new params, one to decide whether `stratify` is required `True` or `False` and if `True` the name of the column on which `stratification` is required. This is applicable to both `pycaret.classification` and `pycaret.regression`.

Hope this will solve the problem.

**Comment by ykh96** [comment]
> @ykh96 That's correct. We are unfortunately forcing `stratify` param of `train_test_split` to target variable and hence `shuffle` is forced to `True`. In `2.2` release we will be introducing two new params, one to decide whether `stratify` is required `True` or `False` and if `True` the name of the column on which `stratification` is required. This is applicable to both `pycaret.classification` and `pycaret.regression`.
> 
> Hope this will solve the problem.

Sounds good. Thanks a lot for your help!

**Comment by pycaret** [comment]
@ykh96 

This issue has been fixed in the `pycaret-nightly` release. Please install pycaret-nightly using `pip install pycaret-nightly`. Feel free to open a new issue, in case the problem persists. This fix will be published to PyPi as `pycaret==2.2.0` on October 28, 2020.
